### PR TITLE
What's new - StackoverflowException fix attempt

### LINKED ISSUE #1314: `StackOverflowException` - `WhatsNewPanel.cs`
> More important will be fixing `StackOverflowException` (one of the last workshop comments - Stable), log: <https://onedrive.live.com/?authkey=%21AG5WxpzcRVvkG9o&cid=7989F98A9950A030&id=7989F98A9950A030%2139983&parId=root&o=OneUp>

_Originally posted by @krzychu124 in https://github.com/CitiesSkylinesMods/TMPE/issues/1312#issuecomment-1021316872_

### PR DESCRIPTION
**Pull request by krzychu124**
Reduced `autolayout` use where possible

Hopefully closes #1314 
[Compiled Zip](https://ci.appveyor.com/api/projects/krzychu124/tmpe/artifacts/TMPE.zip?branch=bugfix/1314-stackoverflow-whats-new)

In debug mode (mono) opening window caused full recalculation of UI, probably also visible in release - notice main menu buttons text for a split second

https://user-images.githubusercontent.com/19638970/153734575-12b30281-88fd-4318-a73f-45f90dbd9f35.mp4

With changes in this PR that issue seem to be solved (tried with full changelog list too) - maybe it's just me but it feels like it opens much faster/with less delay between click and show.

https://user-images.githubusercontent.com/19638970/153734595-e49ca683-ca5a-419d-8d47-b17ce894ffc2.mp4

### CODE REVIEW

File: TLM/TLM/UI/WhatsNew/WhatsNewPanel.cs [hunk 1]
```diff
@@ -79,15 +81,17 @@ public class WhatsNewPanel : UIPanel {
...
             content.autoLayout = true;
+            panel.autoLayout = true;
         }
 
         private void AddChangelogContent(Changelog changelog, UIScrollablePanel uiScrollablePanel) {
             UIPanel panel = AddRowAutoLayoutPanel(parentPanel: uiScrollablePanel,
                                                   panelPadding: new RectOffset(5, 0, 0, 6),
                                                   panelWidth: _defaultWidth - 5,
                                                   vertical: true);
+            panel.minimumSize = new Vector4(_defaultWidth - 10, true ? 46 : 36);
```
Comment (aubergine10) [inline TLM/TLM/UI/WhatsNew/WhatsNewPanel.cs:93]: `true ? 46 : 36` ...?!

**Review by aubergine10 — CHANGES_REQUESTED** [review]

File: TLM/TLM/UI/WhatsNew/WhatsNewPanel.cs (hunk 1 above)
Comment (krzychu124) [reply TLM/TLM/UI/WhatsNew/WhatsNewPanel.cs:93]: shit.. forgot to revert. Good catch! XD

**Review by aubergine10 — APPROVED** [review]
Built from source & tested from main menu / in-game.

LGTM 👍

**Comment by krzychu124** [comment]
Ok, I've got report from the user affected with crash on click - he said it works now 🥳

**Review by chameleon-tbn — APPROVED** [review]
What Else to say... Lgtm

**Pull request merged**